decidim-comments/app/services/decidim/comments/new_comment_notification_creator.rb

```diff
@@ -44,6 +48,16 @@ def notify_mentioned_users
         notify(:user_mentioned, affected_users: affected_users)
       end
 
+      def notify_mentioned_groups
+        mentioned_groups.each do |group|
+          group_users = UserGroups::AcceptedMemberships.for(group).map(&:user)
```

Comment:
the existence of this query object is confusing, many devels could not know its existence when developing other parts of the code.

_If you have the time_, can you refactor and create a `group.accepted_memberships` and `group.accepted_users` methods that ease the use of this query?

Reply:
good idea, I'll do it!